contracts/GSN/IRelayHub.sol

```diff
@@ -141,16 +195,25 @@ contract IRelayHub {
         bytes memory approvalData
     ) public;
 
-    // Emitted when an attempt to relay a call failed. This can happen due to incorrect relayCall arguments, or the
-    // recipient not accepting the relayed call. The actual relayed call was not executed, and the recipient not charged.
-    // The reason field contains an error code: values 1-10 correspond to PreconditionCheck entries, and values over 10
-    // are custom recipient error codes returned from acceptRelayedCall.
+    /**
+     * @dev Emitted when an attempt to relay a call failed.
+     *
+     * This can happen due to incorrect {relayCall} arguments, or the recipient not accepting the relayed call. The
+     * actual relayed call was not executed, and the recipient not charged.
+     *
+     * The `reason` field contains an error code: values 1-10 correspond to `PreconditionCheck` entries, and values
```

Comment:
Is it event parameters rather than fields? https://solidity.readthedocs.io/en/develop/abi-spec.html?highlight=parameter#events

```suggestion
     * The `reason` parameter contains an error code: values 1-10 correspond to `PreconditionCheck` entries, and values
```